packages/jaeger-ui/src/components/Monitor/ServicesView/operationDetailsTable/index.tsx

```diff
@@ -235,12 +242,18 @@ export class OperationTableDetails extends React.PureComponent<TProps, TState> {
               },
             };
           }}
-          onChange={(pagination, filters, { columnKey, order }) => {
-            if (!isEqual({ columnKey, order }, this.state.tableSorting)) {
-              const clickedColumn = tableTitles.get(columnKey || this.state.tableSorting.columnKey);
+          onChange={(pagination, filters, sorter) => {
+            const activeSorters = isArray(sorter) ? sorter : [sorter];
+            const { tableSorting } = this.state;
+
+            if (!isEqual(activeSorters, tableSorting)) {
+              const lastColumn =
+                activeSorters[activeSorters.length - 1] ?? tableSorting[tableSorting.length - 1];
```

Comment:
if both arrays are empty, isn't it possible to end up with `lastColumn = undefined`?

Reply:
But, I think, in that case, the onChange function wouldn't be called.
Whenever someone clicks the table header for sorting, the tableSorting or activeSorters would always return an array of size greater than or equal to 1.